src/perishable-goods/test/logic.feature

```diff
@@ -75,16 +71,15 @@ Formula for Breach Penalty Calculation:
 """
 [
   {
-    "$class": "org.accordproject.cicero.runtime.PaymentObligation",
+    "$class": "org.accordproject.obligation.PaymentObligation",
     "amount": {
       "$class": "org.accordproject.money.MonetaryAmount",
       "doubleValue": 300,
       "currencyCode": "USD"
     },
-    "description": "DAN should pay shipment amount to PETER",
-    "promisor": "resource:org.accordproject.cicero.contract.AccordParty#DAN",
-    "promisee": "resource:org.accordproject.cicero.contract.AccordParty#PETER",
-    "eventId": "valid"
```

Comment:
`eventId` is replaced by `$identifier` now, right?

Reply:
Correct: added by the validator so shows in the output:
```
zsh-5.8$ cd ../payment-upon-delivery/
zsh-5.8$ cicero trigger
6:51:11 AM - INFO: Using current directory as template folder
6:51:11 AM - INFO: Loading a default text/sample.md file.
6:51:11 AM - INFO: Loading a default request.json file.
6:51:13 AM - WARN: A state file was not provided, initializing state. Try the --state flag or create a state.json in the root folder of your template.
6:51:13 AM - INFO: 
{
  "clause": "payment-upon-delivery@0.12.0-ff3f68ca77f73bfa73969884d5fc6be1592154e9aae2ef11a67cb66f87fb0f59",
  "request": {
    "$class": "org.accordproject.payment.upondelivery.DeliveryAcceptedRequest"
  },
  "response": {
    "$class": "org.accordproject.payment.upondelivery.DeliveryAcceptedResponse",
    "$timestamp": "2021-04-14T06:51:13.265-04:00"
  },
  "state": {
    "$class": "org.accordproject.runtime.State",
    "$identifier": "010187ba-ef2b-490f-8836-92dc4be6e1a6"
  },
  "emit": [
    {
      "$class": "org.accordproject.obligation.PaymentObligation",
      "amount": {
        "$class": "org.accordproject.money.MonetaryAmount",
        "doubleValue": 11.98,
        "currencyCode": "USD"
      },
      "description": ""resource:org.accordproject.contract.Party#Michael" should pay cost of goods and delivery fee to "resource:org.accordproject.contract.Party#Diana"",
      "$identifier": "5c3bd7c5-bd2b-48b8-ad5b-ed0d46d24d7b",
      "contract": "resource:org.accordproject.payment.upondelivery.PaymentUponDeliveryContract#2ca304c5-f9f0-4d93-86a9-5095d0a4726c",
      "promisor": "resource:org.accordproject.contract.Party#Michael",
      "promisee": "resource:org.accordproject.contract.Party#Diana",
      "$timestamp": "2021-04-14T06:51:13.266-04:00"
    }
  ]
}
```